_sass/strimzi.scss

```diff
@@ -524,6 +524,16 @@ blockquote::after {
   margin: 0 -13rem;
   padding: 2rem 13rem;
 
+  @media screen and (max-width: 1170px) {
+    margin: 0 -6rem;
+    padding: 2rem 6rem;
+  }
+
+  @media screen and (max-width: 768px) {
+    margin: 0 -2rem;
+    padding: 2rem;
+  }
+
```

Comment:
not sure changing margin and padding based on a specific size in pixels is the way to go today and not based on some percentage and scaling accordingly. But I am not the expert on this kind of stuff.

Reply:
It's there for the Zoom band as well and across other things (nowhere is percentage, only pixels)

Reply:
And also - I'm not CSS expert - but the percentage is calculated against something and I'm not sure against what. Having pixels there worked before and works today as well. I'm just following the way how we have it throughout the all CSS styles.